packages/brokers/src/brokers/redis/BaseRedis.ts

```diff
@@ -23,36 +22,32 @@ export interface RedisBrokerOptions extends BaseBrokerOptions {
 	 * How long to block for messages when polling
 	 */
 	blockTimeout?: number;
-
 	/**
 	 * Consumer group name to use for this broker
 	 *
 	 * @see {@link https://redis.io/commands/xreadgroup/}
 	 */
 	group: string;
-
 	/**
 	 * Max number of messages to poll at once
 	 */
 	maxChunk?: number;
-
 	/**
 	 * Unique consumer name.
 	 *
 	 * @see {@link https://redis.io/commands/xreadgroup/}
 	 */
-	name?: string;
+	name: string;
 }
 
 /**
  * Default broker options for redis
  */
 export const DefaultRedisBrokerOptions = {
 	...DefaultBrokerOptions,
-	name: randomBytes(20).toString('hex'),
 	maxChunk: 10,
 	blockTimeout: 5_000,
-} as const satisfies Required<Omit<RedisBrokerOptions, 'group'>>;
+} as const satisfies Required<Omit<RedisBrokerOptions, 'group' | 'name'>>;
```

Comment:
From a docs perspective it would be better to have name and group be in their own interface and have the options interfaces extend both interfaces and have another interface only extending the one contained in these defaults. Omit is always bad for docs. Apart from that this LGTM